debezium-core/src/main/java/io/debezium/transforms/ExtractNewRecordStateConfigDefinition.java

```diff
@@ -125,7 +127,9 @@ public static DeleteHandling parse(String value, String defaultValue) {
             .withWidth(ConfigDef.Width.LONG)
             .withImportance(ConfigDef.Importance.LOW)
             .withDefault("")
-            .withDescription("Adds each field listed to the header,  __ (or __<struct>_ if the struct is specified) "
-                    + "Example: 'version,connector,source.ts_ms' would add __version, __connector and __source_ts_ms fields");
+            .withDescription("Adds each field listed to the header,  __ (or __<struct>_ if the struct is specified). "
+                    + "Example: 'version,connector,source.ts_ms' would add __version, __connector and __source_ts_ms fields. "
+                    + "Optionally one can also map new field name like version:VERSION,connector:CONNECTOR,source.ts_ms:EVENT_TIMESTAMP."
+                    + "Please note that the new field name is case-insensitive.");
```

Comment:
@rgannu As I pointed in the previous comment, here is the same problem

Reply:
yes.. Fixed it..
https://github.com/debezium/debezium/blob/e539d647cb3091d564d58952ab0218de7185c77b/documentation/modules/ROOT/pages/configuration/event-flattening.adoc

Reply:
Thanks.. Fixed it..